tests/wycheproof/test_hmac.py

```diff
@@ -7,6 +7,7 @@
 import pytest
 
 from cryptography.exceptions import InvalidSignature
+from cryptography.hazmat.bindings._rust import openssl as rust_openssl
```

Comment:
Unused import?

Reply:
Yes poor cleanup from the feedback commit :)